src/runtime-rs/crates/resource/src/network/endpoint/endpoints_test.rs

```diff
@@ -33,6 +65,8 @@ mod tests {
         let dummy_name = format!("dummy{}", idx);
         let vlanid = 123;
 
+        let d = get_device_manager().await.unwrap();
```

Comment:
Although this is a test, I prefer not to use any abbreviation in our code.

Reply:
fixed